src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const smallCashBack = 20;
+  const largeCashBack = 50;
+  const priceDayRentCar = 40;
+  const wholeAmount = days * priceDayRentCar;
+  let priceRentCar = 0;
+
+  if (days >= 3) {
+    priceRentCar = wholeAmount - smallCashBack;
+  }
+
+  if (days < 3) {
+    priceRentCar = wholeAmount;
+  }
+
+  if (days >= 7) {
+    priceRentCar = wholeAmount - largeCashBack;
```

Comment:
Let's reduce the amount of checks. Use only two checks:
if (days < 3) and if (days <=7) or if (days >=7) and if (days >=3)
(in that order)

Also, use `if - else if` construction instead of just many `if`s